app/styles/app.scss

```diff
@@ -87,3 +87,22 @@ body {
 .btn btn-danger, btn-primary, a, hover {
   color: $white;
 }
+
+.item{
+    background: #333;    
+    text-align: center;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered background, height, text-align